sdks/python/apache_beam/dataframe/io.py

```diff
@@ -131,6 +131,11 @@ def _binary_writer(format):
 read_hdf = to_hdf = frame_base.wont_implement_method('random access files')
 
 
+for name in dir(pd):
+  if name.startswith('read_') and name not in globals():
+    globals()[name] = frame_base.not_implemented_method(name)
```

Comment:
Will not_implemented_method work here? It's wrapper takes a `self` parameter

Reply:
Good point. Updated to not require self.